morecantile/defaults.py

```diff
@@ -56,4 +56,4 @@ def register(
         return TileMatrixSets({**self.tms, **new_tms})
 
 
-tms = TileMatrixSets(deepcopy(default_tms))  # noqa
+tms = TileMatrixSets(copy(default_tms))  # noqa
```

Comment:
deepcopy complained about the pyproj Transformer. Used `copy` and it worked 🤷‍♂️